CMake/FindYARA.cmake

```diff
@@ -0,0 +1,51 @@
+# - Find the YARA library
+# This module defines
+#  YARA_INCLUDE_DIR, path to yara.h, etc.
+#  YARA_LIBRARIES, the libraries required to use YARA.
+#  YARA_FOUND, If false, do not try to use YARA.
+# also defined, but not for general use are
+# YARA_LIBRARY, where to find the YARA library.
+
+# Apple readline does not support readline hooks
+# So we look for another one by default
+IF(APPLE)
+  FIND_PATH(YARA_INCLUDE_DIR NAMES yara.h PATHS
+    /sw/include
+    /opt/local/include
+    /opt/include
+    /usr/local/include
+    /usr/include/
+    NO_DEFAULT_PATH)
+ENDIF(APPLE)
+FIND_PATH(YARA_INCLUDE_DIR NAMES yara.h)
+
+IF(APPLE)
+  FIND_LIBRARY(YARA_LIBRARY NAMES yara PATHS
+    /sw/lib
```

Comment:
Is this where fink puts its contents? Do we care to check-- since we're extremely brew-centric.